README.md

```diff
@@ -15,6 +15,7 @@ Since the original repo seems to be abandoned, this fork addresses the following
 
 - [x] NPE when deserializing nullable record attributes of type array
 - [x] deserializing map keys as keywords
+- [x] serializing nullable maps
```

Comment:
I am not sure what you mean my nullable map? Nil values?

Reply:
Yeah, basically, if you marked a field as either `null` or a `map`, it would work only if the map is present (even if it's empty). Passing nil would explode. I've updated the PR. Also, tests now cover this case